src/main/java/iudx/file/server/authenticator/JwtAuthenticationServiceImpl.java

```diff
@@ -84,10 +90,15 @@ public AuthenticationService tokenInterospect(
     String id = authenticationInfo.getString("id");
     String token = authenticationInfo.getString("token");
     String endPoint = authenticationInfo.getString("apiEndpoint");
+
     boolean skipIdCheck =
         endPoint.equalsIgnoreCase(api.getApiFileUpload())
             || endPoint.equalsIgnoreCase(api.getApiFileDelete());
 
+    LOGGER.info("token is" + token);
+    LOGGER.info("the authenticationinfo look slike{}",authenticationInfo);
+    LOGGER.info("the request looks like{}",request);
+
```

Comment:
Don't use info logs for logging user information

Reply:
SO trace should be used?